windows/bleachbit.nsi

```diff
@@ -309,15 +308,10 @@ Section Core (Required)
 
     SetOutPath $INSTDIR
     File /r /x "locale" "..\dist\*.*"
-    File "..\COPYING"
-
-    SetOutPath "$INSTDIR\share\"
-    File "..\bleachbit.png"
```

Comment:
The same question. The COPYING files should not be removed from distribution.

Reply:
Because this is handled from the setup python file. Sorry, probably I had to split this commit and explain better...